.travis.yml

```diff
@@ -20,25 +21,40 @@ matrix:
     - php: 7.0
     - php: 7.1
     - php: 7.2
+    - php: 7.2
+      env:
+        - DEPLOYMENT=STANDALONE_SSL
+    - php: 7.2
+      env:
+        - DEPLOYMENT=SHARDED_CLUSTER
+    - php: 7.2
+      env:
+        - DEPLOYMENT=STANDALONE_AUTH
+    - php: 7.2
+      env:
+        - DEPLOYMENT=REPLICASET
     - php: 7.0
       env:
         - SERVER_VERSION=3.0.15
+        - DEPLOYMENT=STANDALONE_OLD
     - php: 7.0
       env:
         - SERVER_VERSION=3.2.20
+        - DEPLOYMENT=STANDALONE_OLD
     - php: 7.0
       env:
         - SERVER_VERSION=3.4.15
+        - DEPLOYMENT=STANDALONE_OLD
     - php: 7.0
       env:
         - SERVER_VERSION=3.6.5
 
 before_install:
   - pip install "mongo-orchestration>=0.6.7,<1.0" --user `whoami`
   - .travis.scripts/before_install.sh
-  - wget http://fastdl.mongodb.org/linux/mongodb-linux-x86_64-${SERVER_VERSION}.tgz
-  - tar zxf mongodb-linux-x86_64-${SERVER_VERSION}.tgz
-  - export PATH=${PWD}/mongodb-linux-x86_64-${SERVER_VERSION}/bin/:${PATH}
+  - wget http://fastdl.mongodb.org/linux/mongodb-linux-x86_64-ubuntu1404-${SERVER_VERSION}.tgz
+  - tar zxf mongodb-linux-x86_64-ubuntu1404-${SERVER_VERSION}.tgz
+  - export PATH=${PWD}/mongodb-linux-x86_64-ubuntu1404-${SERVER_VERSION}/bin:${PATH}
```

Comment:
Was there a problem with the standard Linux builds? Is it that they ship without SSL support?

I'm just curious if we might run into issues with the build team no longer supplying 14.04 builds for newer releases down the line (assuming Travis doesn't upgrade to 16.04 sooner).

Reply:
Yes, the standard ones ship without SSL support :-/ And yes, we will run into issues when we stop supporting 14.04 and Travis doesn't upgrade :-/